bookwyrm/templates/snippets/privacy_select.html

```diff
@@ -5,16 +5,16 @@
     <label class="is-sr-only" for="privacy-{{ uuid }}">Post privacy</label>
     {% endif %}
     <select name="privacy" id="privacy-{{ uuid }}">
-        <option value="public" {% if not current or current == 'public' %}selected{% endif %}>
+        <option value="public" {% if user.default_post_privacy == 'public' or current == 'public' %}selected{% endif %}>
             Public
         </option>
-        <option value="unlisted" {% if current == 'unlisted' %}selected{% endif %}>
+        <option value="unlisted" {% if user.default_post_privacy == 'unlisted' or current == 'unlisted' %}selected{% endif %}>
             Unlisted
         </option>
-        <option value="followers" {% if current == 'followers' %}selected{% endif %}>
+        <option value="followers" {% if user.default_post_privacy == 'followers' or current == 'followers' %}selected{% endif %}>
             Followers
         </option>
-        <option value="direct" {% if current == 'direct' %}selected{% endif %}>
+        <option value="direct" {% if user.default_post_privacy == 'direct' or current == 'direct' %}selected{% endif %}>
```

Comment:
I think this is going to lead to cases where multiple options will be selected when the `default_post_privacy` is set and the `current` value is set to something different. It should go with `current` if it's set, and use `default_post_privacy` if not